### ISSUE TITLE
How do we properly archive a page?

### ISSUE THREAD

**Issue by vbellissimo** [department-of-veterans-affairs/va.gov-team#59882]
## Issue Description
We need to better understand how to properly archive/sunset a page on the Platform Website. If a page is no longer relevant, or serving a purpose, we need team guidance on how to handle this situation

**Notes:**
- Search for links going to the page that's being archived; How is this done? Waiting on Platform Analytics team for help.
- How do archive in Confluence?
- Other best practices, etc.

---
## Tasks

- [x] Create page in our [Platform Content Team Confluence space](https://vfs.atlassian.net/wiki/spaces/CIT/overview), within the _How we work_ section

- [x] Connect with the rest of the Content Team on steps, guidance taken when needing to sunset a page, (should include steps to take, accounting for instances where a missing link might generate broken links, 404s, etc. if the page is being referenced elsewhere)
- [x] Update Confluence page
- [x] Have team review final edits.

## Acceptance Criteria
- [x] Team consensus on how to handle archiving a page.
- [x] Steps are documented on our team's Confluence Space

**Comment by johnny-jesensky-adhoc** [comment]
As part of the icebox [audit ticket](https://app.zenhub.com/workspaces/platform-content-team-62bde00be46b400017674aef/issues/gh/department-of-veterans-affairs/va.gov-team/69802) in Sprint 41, moving this to the backlog. Will need some additional refinement.

**Comment by jknipes** [comment]
Document completed and located in IA, moving to _Done_
